composer.json

```diff
@@ -1,18 +1,26 @@
 {
 	"name": "hm-base",
-	"license": "GPL V3+",
-	"minimum-stability": "stable",
+	"type": "project",
+	"license": "GPL-3.0-or-later",
+	"require": {
+		"php": "7.0.*",
+		"oomphinc/composer-installers-extender": "*"
+	},
+	"require-dev": {
+		"humanmade/coding-standards": "0.5.0"
+	},
+	"minimum-stability": "dev",
+	"prefer-stable": true,
 	"repositories": [
 		{
 			"type": "composer",
 			"url": "https://wpackagist.org"
 		}
 	],
-	"require-dev": {
-		"humanmade/coding-standards": "0.5.0"
-	},
-	"require": {
-		"oomphinc/composer-installers-extender": "*"
+	"config": {
+		"platform": {
+			"php": "7.0"
```

Comment:
Just a note: here we **do** want to be precise. So, **iff** we were to rely on a specific patch release of PHP, then this should go in here, for example, `"php": "7.0.7"`.

In our case, I think `7.0` is just fine (and then `7.2` sometime soon).

Reply:
I don't think that we can rely on a specific version of PHP for platform purposes on either VIP or our platform due to security updates. Neither platform informs developers of minor version bumps so `x.x` is as close as we can get for sure :/

Reply:
Ah, no. That's no problem. Whatever we put in the platform config is being used to generate the `.lock` file. No more. So we only have to make sure that all systems are running **at least** what we put in the platform config.